### DISCUSSION TITLE
注册时出现问题，帮忙看下这种DEBUG信息显示的问题

### DISCUSSION THREAD

**Discussion by yunkai-jia** [dashingsoft/pyarmor#1376] [Q&A]
INFO     Python 3.9.16
INFO     Pyarmor 8.2.9 (trial), 000000, non-profits
INFO     Platform darwin.x86_64
DEBUG    native platform darwin.x86_64
DEBUG    home path: /Users/yunkai/.pyarmor
INFO     register "/Users/yunkai/Desktop/pyarmor-regfile-5013.zip"
DEBUG    extracting license.lic
DEBUG    extracting .pyarmor_capsule.zip
INFO     update license token
zsh: segmentation fault  pyarmor --debug reg /Users/yunkai/Desktop/pyarmor-regfile-5013.zip

**Comment by jondy** [comment]
请参考 #1333 #1329
